patterns/1-initial/innersource-hackathon.md

```diff
@@ -16,10 +16,10 @@ The company wants to adopt InnerSource as software development methodology but o
 
 The senior leadership believes in InnerSource and wants to drive it throughout the company. The engineers who are familiar with open source principles and/or understand the benefits of InnerSource are the early adopters. There is success with these initial pilot project and teams. Now the next step is to drive it across the company. There might be reluctance from engineering teams due to various factors like:
 
-* not familiar with InnerSource and being ignorant to know about it
+* not familiar with InnerSource and not aware about it
```

Comment:
"not familiar" and "not aware" sounds like saying the same thing twice now. I am not sure though.

I have also asked in [the thread in slack](https://innersourcecommons.slack.com/archives/C2EFRTS6A/p1728330592897019?thread_ts=1727901171.632369&cid=C2EFRTS6A) if anybody has a better proposal.

Reply:
Sorry for the delay in addressing the comments. I have updated this point to "not familiar with InnerSource or open source practices", as suggested by Sally Deering in the slack thread.